config/meltor.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+return [
+
+    // The test run option applies the new migration, and compares the resulting DB structure
+    'testrun' => [
+
+        // This is where the database will be backup up and automatically restored from.
+        'backupFileName'          => 'meltorTestrunBackup.sql',
+
+        // The database structure files which will be the base of comparison.
+        'beforeStructureFileName' => 'meltorStructureBefore.sql',
+        'afterStructureFileName'  => 'meltorStructureAfter.sql',
+    ],
+
+    // MySQL data types are being converted to Laravel fluent ones.
+    'mysql'   => [
+        // https://laravel.com/docs/9.x/migrations#available-column-types
+        'fluentDataTypes' => [
+            'bigint'     => 'bigInteger',
+            // Note that tinyint(1) is being handled separately.
+            'boolean'    => 'boolean',
+            'char'       => 'char',
+            'date'       => 'date',
+            'int'        => 'integer',
+            'json'       => 'json',
+            'longtext'   => 'longText',
+            'mediumtext' => 'mediumText',
+            'text'       => 'text',
+            'time'       => 'time',
+            'timestamp'  => 'timestamp',
+            'tinyint'    => 'tinyInteger',
+            'varchar'    => 'string',
+        ],
+
+        'fluentIntegerTypes' => [
```

Comment:
Why are the integrer types stated here again?

Reply:
its the numeric ones, but i have to check if i really need the key-value pair

Reply:
removed the value and kept the keys for integer types

Reply:
but yeah, i ll try to implement all these types

Reply:
added support for a lot of data types. full support is currently out of scope.